test/EFCore.PG.FunctionalTests/BatchingTest.cs

```diff
@@ -182,9 +177,9 @@ public void Inserts_are_batched_only_when_necessary(int minBatchSize)
                         context.SaveChanges();
 
                         Assert.Contains(minBatchSize == 3
```

Comment:
Can this be handled as a `[Theory]`?

(Just skimming the diff, so disregard if it's already parameterized.)

Reply:
On a more general note, for the tests we copy from EF Core I make them as similar as possible to the original, even if some improvements are possible - there's more value in conforming to the original. But of course we can always submit PRs to improve the original.